consensus/scheme/rolldpos/fsm.go

```diff
@@ -456,12 +460,17 @@ func (m *cFSM) handleStartRoundEvt(_ fsm.Event) (fsm.State, error) {
 			Msg("error when getting the proposer")
 		return sInvalid, err
 	}
-	m.ctx.round = roundCtx{
-		height:           height,
-		timestamp:        m.ctx.clock.Now(),
-		proposalEndorses: make(map[hash.Hash32B]map[string]bool),
-		lockEndorses:     make(map[hash.Hash32B]map[string]bool),
-		proposer:         proposer,
+	if m.ctx.round.height == height {
+		m.ctx.round.number = m.ctx.round.number + 1
```

Comment:
Does it matter if different nodes have different view of round?

Reply:
It doesn't matter. The rounds just help more nodes reach consensus.